djangosaml2/views.py

```diff
@@ -340,7 +340,7 @@ def assertion_consumer_service(request,
                              create_unknown_user=create_unknown_user)
     if user is None:
         logger.warning("Could not authenticate user received in SAML Assertion. Session info: %s", session_info)
-        raise PermissionDenied
+        return fail_acs_response(request, exception=Exception('No user could be authenticated.'))
```

Comment:
can we do better than raising the generic `Exception` exception?

Reply:
We can make our own exception class. But for now, perhaps I'll just re-use the permissiondenied one? I'm working in another branch on making the views class-based, which restructures some things, perhaps it's ok to wait until then to revisit this.

Reply:
I've used a PermissionDenied for now